ddtrace/tracer/option.go

```diff
@@ -829,3 +829,42 @@ func StackFrames(n, skip uint) FinishOption {
 		cfg.SkipStackFrames = skip
 	}
 }
+
+// UserTagOption represents a function that can be provided as a parameter to SetUser.
+// This is used to normalize user tags key/value pairs.
+type UserTagOption func() (key string, val interface{})
+
+// WithUserEmailTag emits the normalized key/value UserTagOption holding user email information.
+func WithUserEmailTag(email string) UserTagOption {
```

Comment:
Similarly to my previous comment, you should remove this `Tag` suffix and rename it `WithUserEmail`.
The same applies to your other options 🙏